src/Adapters/Auth/twitter.js

```diff
@@ -5,6 +5,9 @@ var logger = require('../../logger').default;
 
 // Returns a promise that fulfills iff this user id is valid.
 function validateAuthData(authData, options) {
+  if(!options) {
+    throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND, 'Twitter auth configuration missing');
```

Comment:
Yea this will work as well. I took a quick look at all the error codes seeing if there was something more relevant for a misconfig, nothing looks to fit the bill. -1 it is!